crates/executor/src/eth_dao_fork.rs

```diff
@@ -3,10 +3,10 @@
 use reth_primitives::{hex_literal::hex, H160};
 
 /// Dao hardfork beneficiary that received ether from accounts from DAO and DAO creator children.
-pub const DAO_HARDFORK_BENEFICIARY: H160 = H160(hex!("bf4ed7b27f1d666546e30d74d50d173d20bca754"));
+pub static DAO_HARDFORK_BENEFICIARY: H160 = H160(hex!("bf4ed7b27f1d666546e30d74d50d173d20bca754"));
 
 /// DAO hardfork account that ether was taken and added to beneficiry
-pub const DAO_HARDKFORK_ACCOUNTS: [H160; 116] = [
+pub static DAO_HARDKFORK_ACCOUNTS: [H160; 116] = [
```

Comment:
`const` is maybe better as it is used only in one place but it does not matter at all, was just curious you changed it :D

Reply:
I thought I have to leave a note on this in this PR but I missed it.

I'd like to change it because `static` is `there is only one instance for each value, and it’s at a fixed location in memory.`, on the other hands `const` is inlined. `DAO_HARDFORK_ACCOUNTS` is a little bit big array and it's frequently iterated for execution. If it's inlined as a `const`, it might be constructed whenever it's used. So I think `static` is better in terms of performance.

ref: https://web.mit.edu/rust-lang_v1.25/arch/amd64_ubuntu1404/share/doc/rust/html/book/first-edition/const-and-static.html

Reply:
I'm little big confusing whether `const` is construct only once or not if `const` is used only in one place. If `const` is constructed only once in this case, I'll revert my change in order to minimize unnecessary changes.

I'm a newbie for reth and also rust. I'm still learning and now I'm reading more material.

ref:
* https://stackoverflow.com/questions/52751597/what-is-the-difference-between-a-constant-and-a-static-variable-and-which-should
* https://github.com/rust-lang/rfcs/blob/master/text/0246-const-vs-static.md

Reply:
Don't worry too much it does not matter, static is fine. as you said diff between const and static is where it is inside the program, for const compiler will put it in code and do whatever the compiler does, static would create this once in runtime and share the reference

Reply:
I wouldn't worry about this and we can evaluate if this is an issue in the future